parent/pom.xml

```diff
@@ -85,6 +85,8 @@
    <properties>
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <plugins.chm/>
+      <plugins.couchbase/>
+      <plugins.couchbase-latest/>
```

Comment:
why is this needed?

Reply:
I believe this is used to discover the plugin projects and is used by the build scripts when creating the distributable ZIP.

Reply:
First of all, thank you for this contribution. This property allows you to build a RadarGun distribution that only includes the latest version of each plugin, instead of every version of the plugin.